Enterprise-CMCS/macpro-mdct-qmr
# Tech Spec Changes - COL-HH

### Description
1. Add **Electronic Clinical Data Systems (ECDS)** data source to "Data Source" section of the COL-HH - Colorectal Cancer Screening form

2. Text changes to Performance Measure description of the COL-HH - Colorectal Cancer Screening form


### Related ticket(s)
MDCT-https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2373

---
### How to test
1. Login using the stateuser4@test.com user
2. Add Health Home data
3. Select Health Home core set name
4. Scroll down to select COL-HH Colorectal Cancer Screening measure
5. Scroll down to **Data Source** section (This should contain the new option ECDS)
6. Question - "Are you reporting on this measure?" select yes
7. Scroll to **Measurement Specification** section - Select any NCQA/HEDIS dropdown option
8. Scroll to **Performance Measure** section to see the text changes.


### Important updates
N/A


---
### Author checklist


---
